basic/03a_command_position.cpp

```diff
@@ -68,6 +72,17 @@ int main() {
 
   // Stop logging
   auto log_file = group->stopLog();
-
+  std::vector<std::vector<double>> pos;
```

Comment:
It'd be nice to have a comment here describing what we're doing (e.g., "plot position from the log")

Also, I added a planned feature for our C++ API to wrap some of this in the future to make it easier to get data from log files.